src/compat_wrappers.h

```diff
@@ -8,6 +8,10 @@
 
 #include "b-em.h"
 
+#ifndef WIN32
+#define HAVE_STPCPY
+#endif
```

Comment:
At the moment the MingW build is using a normal makefile not autotools.  Perhaps this should just be an inversion of what is currently there, i.e.
`#ifdef WIN32
#undef HAVE_STPCPY
#endif`

Related to that, what's the latest on -Werror and the test for stpcpy?  At the moment it is checked with the macro:

AC_CHECK_FUNCS(tdestroy stpcpy)

but if you remember that generates a C test program that generates warnings from the compiler.  When -Werror is on, and the same flags are used for the test programs as for the actual compilation of the project, those warnings become errors and the test program fails even on Linux with glibc which has the function.